plugins/filter_kubernetes/kube_meta.c

```diff
@@ -111,7 +168,12 @@ static int get_local_pod_info(struct flb_kube *ctx)
     /* If a namespace was recognized, a token is mandatory */
     ret = file_to_buffer(ctx->token_file, &tk, &tk_size);
     if (ret == -1) {
-        flb_plg_warn(ctx->ins, "cannot open %s", FLB_KUBE_TOKEN);
+        flb_plg_warn(ctx->ins, "cannot open %s; running command to get token", FLB_KUBE_TOKEN);
```

Comment:
I want to discuss a bit about this logic. 
So as I understand, if customer setup the config for using command to get token,
we should treat as a decision from customer that the token will be get from this command no matter if the token file exist or not. 
I know for now we are working this for a specific use case and this could work.
But in the future there might be a case when customer has the token file but still would like to use a command to get the token(even though not very likely).
In current logic, it is possible that customer setup the command but the token are finally still from token file.
So my suggestion would be:
if run command is setup, we should skip the token file load and use the command directly.

Reply:
My thought is that we can treat the token file as the priority. If we don't have it, we can try to find if our customers have provided a command for us to run. as token file is our primary way to get token..? at least for now

Reply:
good point to discuss.

Reply:
I agree with Drew's logic. Its solid. These options should be mutually exclusive. Like you should not be able to configure both `Token_Command` and `Kube_Token_File`. Its one or the other; user picks. That makes it much less confusing.